Source/WebKit/Shared/haiku/WebCoreArgumentCodersHaiku.cpp

```diff
@@ -38,47 +38,47 @@ using namespace WebCore;
 
 namespace IPC {
 
-void ArgumentCoder<ResourceRequest>::encodePlatformData(ArgumentEncoder& encoder, const ResourceRequest& resourceRequest)
+void ArgumentCoder<ResourceRequest>::encodePlatformData(Encoder& encoder, const ResourceRequest& resourceRequest)
 {
     notImplemented();
 }
 
-bool ArgumentCoder<ResourceRequest>::decodePlatformData(ArgumentDecoder& decoder, ResourceRequest& resourceRequest)
+bool ArgumentCoder<ResourceRequest>::decodePlatformData(Decoder& decoder, ResourceRequest& resourceRequest)
 {
     notImplemented();
     return false;
 }
 
-void ArgumentCoder<ResourceResponse>::encodePlatformData(ArgumentEncoder& encoder, const ResourceResponse& resourceResponse)
+/*void ArgumentCoder<ResourceResponse>::encodePlatformData(Encoder& encoder, const ResourceResponse& resourceResponse)
```

Comment:
this one doesnt have a matching template in webcoderarguement. so shall i add a template for resourceresponse ? i suppose other platforms dont use resourceresponse so i commented it .

Reply:
There is a resourceError template i think it covers response? what say